tests/fsharpqa/Source/Conformance/ObjectOrientedTypeDefinitions/ClassTypes/MemberDeclarations/E_ClashingInstanceStaticProperties.fs

```diff
@@ -1,6 +1,6 @@
 // #Regression #Conformance #ObjectOrientedTypes #Classes #MethodsAndProperties #MemberDefinitions 
 // Regression test for FSHARP1.0:5475
-//<Expects status="error" span="(5,27-5,28)" id="FS0441">Duplicate property\. The property 'M' has the same name and signature as another property in type 'C'\.$</Expects>
+//<Expects status="error" span="(6,31-6,32)" id="FS0441">Duplicate property\. The property 'M' has the same name and signature as another property in type 'C'\.$</Expects>
 type C = class
```

Comment:
Why did this range change?  Although it seems as if it is now correct.

Reply:
Yes, it's a better range.  It will have to do with ordering of keys in the dictionary, which we should not depend on.

Reply:
@dsyme, that was my theory too.  I will add an issue.  But we don't have to fix it now.